datadog/resource_datadog_synthetics_test_.go

```diff
@@ -295,6 +303,44 @@ func syntheticsTestOptions() *schema.Schema {
 	}
 }
 
+func syntheticsTestOptionsList() *schema.Schema {
+	return &schema.Schema{
+		Type:          schema.TypeList,
+		Optional:      true,
+		MaxItems:      1,
+		ConflictsWith: []string{"options"},
```

Comment:
nth: As discussed earlier, we should add a test if `ConflictsWith` can raise an error